docker/dev/docker-compose.yml

```diff
@@ -48,20 +48,14 @@ services:
       "
 
   xain-fl-dev:
-    environment:
-      MINIO_ACCESS_KEY: minio
-      MINIO_SECRET_KEY: minio123
-    build:
-      context: .
-      dockerfile: Dockerfile.dev
-    command: sh -c "coordinator -f test_array.npy --storage-endpoint http://minio-dev:9000 --storage-key-id $${MINIO_ACCESS_KEY} --storage-secret-access-key $${MINIO_SECRET_KEY} --storage-bucket xain-fl-aggregated-weights"
+    image: xain-fl-dev
```

Comment:
The idea of the "dev" image is to build and run locally, not to pull from a registry. By removing the `build` directive, an image by the name of "xain-fl-dev" will try to be pulled from docker hub

Reply:
Ah yes, I did that temporarily because I wanted to docker-compose to reuse the local image I had already built with `docker build`. I forgot to undo.

Reply:
I reverted this change.